miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplayTest.kt

```diff
@@ -38,9 +43,9 @@ class RealMiniAppDisplayTest {
         }
 
     @Test
-    fun `when getLoadUrl is called then a formed path is returned from basePath for loading`() {
+    fun `when getLoadUrl is called then a formed path contains app id`() {
         val realDisplay = getRealMiniAppDisplay()
-        realDisplay.getLoadUrl() shouldBeEqualTo "file://dummy/index.html"
+        realDisplay.getLoadUrl() shouldContain TEST_MA_ID
```

Comment:
Maybe this should be testing that the URL points to `index.html`? We're already testing above that the path contains the App Id.